recipe/build.sh

```diff
@@ -1,38 +1,19 @@
 #!/bin/bash
 
-args="-DOVITO_BUILD_PLUGIN_MESH=ON \
-      -DOVITO_BUILD_PLUGIN_PARTICLES=ON \
-      -DOVITO_BUILD_PLUGIN_TACHYON=OFF \
-      -DOVITO_BUILD_PLUGIN_CRYSTALANALYSIS=OFF \
-      -DOVITO_BUILD_PLUGIN_NETCDFPLUGIN=OFF \
-      -DOVITO_BUILD_PLUGIN_POVRAY=OFF \
-      -DOVITO_BUILD_PLUGIN_OPENBABELPLUGIN=OFF \
-      -DOVITO_BUILD_PLUGIN_VR=OFF \
-      -DOVITO_BUILD_PLUGIN_CORRELATION=ON \
-      -DOVITO_BUILD_PLUGIN_VOROTOP=ON \
-      -DOVITO_BUILD_GUI=OFF \
-      -DOVITO_BUILD_DOCUMENTATION=OFF \
-      -DCMAKE_BUILD_TYPE=Release \
-      -DCMAKE_INSTALL_PREFIX=${PREFIX}" 
-
 mkdir build
 cd build
-if [ ${PY3K} = 1 ]; then 
-	cmake $args \
-	      -DPYTHON_EXECUTABLE=${PREFIX}/bin/python \
-	      -DPYTHON_INCLUDE_DIR=${PREFIX}/include/python${PY_VER}m \
-	      -DPYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}m${SHLIB_EXT} \
-	      ..
-else 
-	cmake $args \
-	      -DPYTHON_EXECUTABLE=${PREFIX}/bin/python \
-	      -DPYTHON_INCLUDE_DIR=${PREFIX}/include/python${PY_VER} \
-	      -DPYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}${SHLIB_EXT} \
-	      ..
-fi;
-make 
-cp lib/ovito/*${SHLIB_EXT} ${PREFIX}/lib
-cp -r lib/ovito/plugins/python/ovito ${SP_DIR}
-mkdir -p ${PREFIX}/lib/ovito/plugins
-cp lib/ovito/plugins/*${SHLIB_EXT} ${PREFIX}/lib/ovito/plugins
-sed -i 's\/../../..\/../../../../ovito/plugins\g' ${SP_DIR}/ovito/plugins/__init__.py
+
+cmake -DOVITO_BUILD_DOCUMENTATION=ON \
+      -DOVITO_BUILD_GUI=ON \
+      -DOVITO_BUILD_MONOLITHIC=OFF \
+      -DOVITO_REDISTRIBUTABLE_PACKAGE=OFF \
+      -DOVITO_USE_PRECOMPILED_HEADERS=ON \
+      -DOVITO_BUILD_PYTHON_PACKAGE=OFF \
```

Comment:
Adding this here as future reference. With the release of ovito 3.X the python interface is part of ovito pro, therefore it is no longer included here: https://www.ovito.org/about/ovito-pro-features/

Reply:
Yes we shall also need to update the description field to explain that this is "Ovito Basic".

Reply:
Sure feel free to make all the changes you need. Once you are ready I can take some time to review them.